src/docs/customize/global-props.mdx

```diff
@@ -0,0 +1,104 @@
+---
+name: Global Props
+menu: Customize
+route: /customize/global-props
+---
+
+# Global Props
+
+import { Playground } from 'docz'
+import {
+  Toolbar,
+  ToolbarItem,
+} from '../../lib/components/layout/Toolbar';
+import Button from '../../lib/components/ui/Button';
+import SelectField from '../../lib/components/ui/SelectField';
+import TextField from '../../lib/components/ui/TextField';
+import { RUIProvider } from '../../lib/provider';
+
+Sometimes it can be useful to share some configuration props between
+multiple instances of components of the same or different kind.
+
+This can be achieved by wrapping application or its part with `RUIProvider`
+and passing in the common props through `globalProps` prop. With or without
+using `RUIProvider` the component can be configured by passing in the props
+directly. The props passed in directly always take precedence over props
+coming from `RUIProvider`.
```

Comment:
```suggestion
This can be achieved by wrapping application or its part with `RUIProvider`
and passing in common props through the `globalProps` prop. Regardless of
using `RUIProvider`, the component can still be configured by passing in the props
directly. The props passed in directly always take precedence over props
coming from `RUIProvider`.
```

Just my feelings, use it or not as you see fit.